### ISSUE TITLE
IE9 Object doesn't support property or method 'assign' 

### ISSUE THREAD

**Issue by laxplaer** [akiran/react-slick#156]
IE9 Object doesn't support property or method 'assign' 

```javascript
slides.push(_react2['default'].cloneElement(child, {
      key: index,
      'data-index': index,
      className: getSlideClasses((0, _objectAssign2['default'])({ index: index }, spec)),
      style: Object.assign({}, child.props.style || {}, childStyle)
}));
```

To be more concrete it is only in 0.9.0-rc1

**Comment by le0pard** [comment]
Have the same issue

**Comment by websilone** [comment]
IE10 doesn't support it either.

Using babel already, I pulled in the babel polyfill which handles this.
It's working fine now (on IE10 at least, didn't try it yet on IE9).

**Comment by laxplaer** [comment]
Thank you for the tip about that polyfill. I have used https://github.com/paulmillr/es6-shim and it works just fine.

**Comment by smwhit** [comment]
slides.push(_react2['default'].cloneElement(child, {
      key: index,
      'data-index': index,
      className: getSlideClasses((0, _objectAssign2['default'])({ index: index }, spec)),
      style: Object.assign({}, child.props.style || {}, childStyle)
}));

I think Object.assign should be replaced with _objectAssign

**Comment by akiran** [comment]
Object.assign issue is fixed in react-slick@0.9.1

**Comment by alexkrolick** [comment]
Doesn't look fixed on 0.12.5
